zerver/tests/test_subs.py

```diff
@@ -3257,7 +3257,10 @@ def test_user_settings_for_adding_public_streams(self) -> None:
         )
 
     def _test_user_settings_for_creating_streams(
-        self, stream_policy: str, invite_only: bool
+        self,
+        stream_policy: str,
+        invite_only: bool,
```

Comment:
Should this add a `, *, ` after `stream_policy` to enforce always specifying these last two parameters.

Reply:
Done.